platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpLogoutActionProvider.java

```diff
@@ -56,7 +56,9 @@
             try {
 
                 @SuppressWarnings("unchecked")
-                String nameId = SubjectUtils.getName((Subject) ((Map) realmSubjectMap).get("idp"));
+                String nameId = SubjectUtils.getName((Subject) ((Map) realmSubjectMap).get("idp"),
```

Comment:
SubjectUtils already has unit tests to cover when the name can't be found and it falls back on the default value, which is "You" in this case.